lib/registration.pm

```diff
@@ -229,17 +229,23 @@ Wrapper for SUSEConnect -p $name  over ssh.
 =cut
 
 sub ssh_add_suseconnect_product {
-    my ($remote, $name, $version, $arch, $params, $timeout, $retries, $delay) = @_;
+    my ($remote, $name, %args) = @_;
```

Comment:
Is documentation at https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20942/files#diff-e5ae008bc24ec64aac7d7f80f7671324b746ffe329c8fd6c8e5dba662f942ae7R226 still ok after this change?